src/LudoGameEngine/Game.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+namespace LudoGameEngine
+{
+    public class Game
+    {
+        private Player[] Players;
+        public int NrOfPlayers;
```

Comment:
What happens if this is updated during the game? Can that be avoided?

Reply:
Suggest either setting number of players in the constructor, or have a method for it e.g. AddPlayer(...)
Either way, a change in the number of players is something the Game-class needs to react to (or disallow) so NrOfPlayers should not be directly settable by any code outside of the Game-class, i.e. `public int PlayerCount {get; private set;}` (it is more common to use "...Count" as opposed to "NumberOf..." or similar)